prepare/cards/mt_bench_judge.py

```diff
@@ -0,0 +1,13 @@
+from unitxt.blocks import TaskCard
+from unitxt.catalog import add_to_catalog
+
+card = TaskCard(
+    loader=None,
+    preprocess_steps=[],
+    task="tasks.llm_as_judge.rag.model_response_assessment",
```

Comment:
Not sure we need llm_as_judge prefix.  This is because same tasks can be implemented with LLM or not.  E.g. answer_correctness.